distributed/tests/test_actor.py

```diff
@@ -465,7 +465,7 @@ def f(block, ps=None):
     print(format_time(end - start))
 
 
-@pytest.mark.flaky(reruns=10, reruns_delay=5)
+@pytest.mark.flaky(max_runs=10)
```

Comment:
We use [`pytest-rerunfailures`](https://github.com/pytest-dev/pytest-rerunfailures) for handling flaky tests, which doesn't have a `max_runs` parameter. Perhaps you're thinking of [`flaky`](https://github.com/box/flaky) (another project for handling flaky tests) which unfortunately also registers a `flaky` marker which does have a `max_runs` parameter. 

```suggestion
@pytest.mark.flaky(reruns=10, reruns_delay=5)
```